composeApp/src/commonMain/kotlin/com/spotride/mobile/UserApiService.kt

```diff
@@ -0,0 +1,31 @@
+package com.spotride.mobile
+
+import io.ktor.client.HttpClient
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.logging.LogLevel
+import io.ktor.client.plugins.logging.Logging
+import io.ktor.client.request.get
+import io.ktor.client.statement.bodyAsText
+import io.ktor.serialization.kotlinx.json.*
+import kotlinx.serialization.json.Json
+
+class UserApiService {
+    private val url = "http://10.0.2.2:8080/users"
```

Comment:
localhost?

Reply:
Yes, but it is specific address for android emulator